recipe/cython-scoping.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/healpy/src/_line_integral_convolution.pyx b/healpy/src/_line_integral_convolution.pyx
+index 12ba874..aefd451 100644
+--- a/healpy/src/_line_integral_convolution.pyx
++++ b/healpy/src/_line_integral_convolution.pyx
+@@ -2,7 +2,7 @@ import numpy as np
+ cimport numpy as np
+ cimport cython
+ from _common cimport RING, Healpix_Map, ndarray2map
+-from pixelfunc import npix2nside
++from healpy.pixelfunc import npix2nside
```

Comment:
Does this work?
```suggestion
-from pixelfunc import npix2nside
+from .pixelfunc import npix2nside
```

Reply:
I would think that it would, but some of the other `.pyx` files use the absolutely-scoped syntax, so I copied them and didn't test the other form.